release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -436,6 +436,11 @@ You can also customize the pod lifetime value for the `LifecycleAndUtilization`
 
 For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting pods using the descheduler].
 
+[id="ocp-4-9-node-health-check-operator"]
+==== Deploy node health checks with the Node Health Check Operator (Technology Preview)
+
+You can use the Node Health Check Operator to deploy the NodeHealthCheck controller. The controllers identifies unhealthy nodes and uses the Poison Pill Operator to remediate the unhealthy nodes.
```

Comment:
I would change "The controllers identifies" to "The controllers identify". Unless there is only one controller deployed, which the previous line seems to indicate,  then I would think it would read "The controller identifies"

Reply:
@kquinn1204 Thank you. Sorry, that was a typo. I have updated it to "The controller identifies.."

Reply:
LGTM